ros_gz_sim/launch/ros_gz_sim.launch.py

```diff
@@ -13,59 +13,100 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-"""Launch gzsim in a component container."""
+"""Launch gzsim + ros_gz_bridge in a component container."""
 
-import launch
-from launch.actions import DeclareLaunchArgument
-from launch.substitutions import LaunchConfiguration, TextSubstitution
-from launch_ros.actions import ComposableNodeContainer
-from launch_ros.descriptions import ComposableNode
+from launch import LaunchDescription
+from launch.actions import DeclareLaunchArgument, IncludeLaunchDescription
+from launch.launch_description_sources import PythonLaunchDescriptionSource
+from launch.substitutions import LaunchConfiguration, PathJoinSubstitution, TextSubstitution
+from launch_ros.substitutions import FindPackageShare
 
 
 def generate_launch_description():
 
-    world_sdf_file_arg = DeclareLaunchArgument(
+    config_file = LaunchConfiguration('config_file')
+    container_name = LaunchConfiguration('container_name')
+    namespace = LaunchConfiguration('namespace')
+    use_composition = LaunchConfiguration('use_composition')
+    use_respawn = LaunchConfiguration('use_respawn')
+    log_level = LaunchConfiguration('log_level')
```

Comment:
I imagine we'll have a `log_level` parameter for gz_sever as well at some point. How would we distinguish parameters for the server and parameters for the bridge? Should we rename this to `bridge_log_level`?

Reply:
Renamed in [66e74f8](https://github.com/gazebosim/ros_gz/pull/533/commits/66e74f8fbe11cdd2412ba36d79ad827d5182190c)